apps/emqx/include/emqx_trace.hrl

```diff
@@ -32,7 +32,8 @@
     payload_encode = text :: hex | text | hidden | '_',
     extra = #{} :: map() | '_',
```

Comment:
put `formatter` here for backward compatibility.

Reply:
Thanks, fixed: [c163956](https://github.com/emqx/emqx/pull/12863/commits/c163956d08f05249dc833dbc2852965ca1799870)